settings.py

```diff
@@ -141,4 +141,4 @@
 TEMPLATE_DIR = os.path.join(CERT_PRIVATE_DIR, TEMPLATE_DATA_SUBDIR)
 
 with open(os.path.join(CERT_PRIVATE_DIR, CERT_DATA_FILE)) as f:
-    CERT_DATA = yaml.load(f.read())
+    CERT_DATA = yaml.load(f.read().decode("utf-8-sig"))
```

Comment:
@marcore we tend to use the `utf-8` encoding throughout the edx codebase. I don't know the risks of using `utf-8-sig` encoding, so I think I would prefer just `utf-8` unless there's a really compelling reason to use a different encoding in just this one location.